src/main/java/com/github/dockerjava/api/model/Ports.java

```diff
@@ -70,7 +71,7 @@ public void bind(ExposedPort exposedPort, Binding binding) {
             if (binding == null) {
                 ports.put(exposedPort, null);
             } else {
-                ports.put(exposedPort, new Binding[] { binding });
+                ports.put(exposedPort, new Binding[]{binding});
```

Comment:
1) It default in my idea and 2) it was used in other places. Only ~4 places was affected.
 If you want i can change to other rule.